calibPipe.py

```diff
@@ -0,0 +1,67 @@
+
+import MySQLdb
```

Comment:
Why not `lsst.db`? I thought it should be easier to use and configure than mysqldb?